### PR TITLE
Updating Dev Hub content - signed in view

### LINKED ISSUE #10309: Update Dev Hub content - signed in view
This is a regularly scheduled update to the Dev Hub (signed in view). 

**Current view:** 

screen shot 2019-01-04 at 12 29 36 pm

**Updates:**
- Remove "create an appealing listing" text & call to action
- Move the right hand "how to get your extension featured" text and call-to-action to the left column 
- Add text to the new right hand section: "You can help users gain confidence and trust in your extension by following our best practices for onboarding users. "
- Add new CTA for right hand section: "See Best Practices " (link to: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/onboarding_upboarding_offboarding_best_practices)

### PR DESCRIPTION
**Pull request by Passionatecricketer**
Fixes #10309

### CODE REVIEW

**Comment by Passionatecricketer** [comment]
@championshuttler Please have a look at it

**Comment by championshuttler** [comment]
@Passionatecricketer also it would be great if you mention something like `Fixes #ISSUENUM` in first comment by editing it , it would help us to close the issue automatically when the PR will be merged.

**Comment by Passionatecricketer** [comment]
@championshuttler Thanks for the suggestions.
I have committed and merged the changes as you suggested.
Please have a look at it and let me know

File: src/olympia/devhub/templates/devhub/new-landing/components/my-addons.html [hunk 1]
```diff
@@ -25,23 +25,23 @@ <h2>{{ _("Welcome Back") }}</h2>
...
-          {% trans %}
-          Help users discover your extension by creating an appealing product page.
+          {% trans amo_link_open='<a href="https://addons.mozilla.org/">'|safe, amo_link_close='</a>'|safe %}
+          Want your extension to be featured on {{ amo_link_open }}addons.mozilla.org{{ amo_link_close }}?
+          Our featured extensions policies will tell you what you need to know.
           {% endtrans %}
         </p>
-        <a href="https://developer.mozilla.org/Add-ons/Listing" class="Button">
+        <a href="https://developer.mozilla.org/Add-ons/AMO/Policy/Featured" class="Button">
           {{ _('Learn How') }}
```
Comment (championshuttler) [inline src/olympia/devhub/templates/devhub/new-landing/components/my-addons.html]:
```suggestion
          {{ _('Learn More') }}
```

@Passionatecricketer We need to change it to `Learn More` as well.

**Comment by Passionatecricketer** [comment]
@championshuttler Sorry I missed it earlier!!
I have made the suggested changes and merged it now.

**Comment by championshuttler** [comment]
> @championshuttler Sorry I missed it earlier!!
> I have made the suggested changes and merged it now.

@Passionatecricketer Awesome 💯

**Comment by caitmuenster** [comment]
Thanks so much, @Passionatecricketer! Your contribution has been added to our [recognition wiki.](https://wiki.mozilla.org/Add-ons/Contribute/Recognition)

Would you be interested in creating a profile on mozillians.org? I'd be happy to vouch for you!

**Comment by Passionatecricketer** [comment]
@caitmuenster Thank you for the recognition and I will be vey happy to be a part of mozilla group and be a mozillian!

**Comment by Passionatecricketer** [comment]
@caitmuenster Here's my profile link 
https://mozillians.org/en-US/u/Rishu4398/ 
and a heartily thanks for the opportunity given.

**Pull request merged**